apps/graphql/src/apps/itinerary/queries/__tests__/__snapshots__/CheckItineraryQueries.test.js.snap

```diff
@@ -0,0 +1,35 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`matches expected output: CheckItinerary.graphql 1`] = `
+~~~~~~~~~~ INPUT ~~~~~~~~~~
+fragment RouteStopFragment on RouteStop {
+  time {
+    utc
+    local
+  }
+}
+
+{
+  checkItinerary(input: { bookingToken: "token", bags: 1, passengers: { adults: 1 }}) {
+    id
+    isValid
+    isChecked
+  }
+}
+
+
+~~~~~~~~~~ OUTPUT ~~~~~~~~~~
+{
+  "errors": [
+    {
+      "message": "Fragment \\"RouteStopFragment\\" is never used.",
```

Comment:
You should fix the query, IMHO this error is not expected

Reply:
🤦‍♂️ Sorry, didn't checked new snapshot. Fixed, updated.